diesel/src/query_source/mod.rs

```diff
@@ -16,15 +16,104 @@ use types::{FromSqlRow, HasSqlType};
 pub use self::joins::JoinTo;
 pub use self::peano_numbers::*;
 
-/// Trait indicating that a record can be queried from the database. This trait
-/// can be derived automatically using `#[derive(Queryable)]`. This trait can
-/// only be derived for structs, not enums.
+/// Trait indicating that a record can be queried from the database.
+///
+/// Types which implement `Queryable` represent the result of a SQL query. This
+/// does not necessarily mean they represent a single database table.
+///
+/// This trait can be derived automatically using `#[derive(Queryable)]`. This
```

Comment:
Should we mention `#[macro_use] extern crate diesel;` anywhere?

Reply:
I think we can safely assume that is always present at this level